### PR TITLE
Fix fields frontend

### PR DESCRIPTION
**Pull request by adriannadiaz**
There were problems, we had different names for the fields in participant, plus we were not saving stuff like telephone. Anna added the fields in the backend, so this incorporates those changes into the frontend and also gives a uniform look to the application. Blank fields are not displayed.

### CODE REVIEW

**Comment by adriannadiaz** [comment]
screen shot 2018-01-12 at 10 35 20 pm

File: angular-client/src/app/components/profiles/profiles.component.css [hunk 1]
```diff
@@ -3,8 +3,9 @@
 }
 
 .example-header-image {
-  background-image: url('http://profile.actionsprout.com/default.jpeg');
+  background-image: url('https://orig00.deviantart.net/dca3/f/2012/006/0/5/princess_luna_facebook_profile_picture_season_2_by_wisami-d4lgj08.jpg');
   background-size: cover;
```
Comment (Ldhatter) [inline angular-client/src/app/components/profiles/profiles.component.css]: i cant really identify what this is, but it's nicer than the nondescript icon previously.

File: angular-client/src/app/components/new-participant/new-participant.component.html [hunk 2]
```diff
@@ -13,13 +8,26 @@
...
       <input type="tel" matInput placeholder="Telephone" name="telephone" [(ngModel)]="participantData.telephone">
     </mat-form-field>
 
     <mat-form-field>
-      <input matInput placeholder="Social media handle" name="socialmedia" [(ngModel)]="participantData.socialmedia">
+      <input matInput placeholder="Social media provider (Twitter, Facebook)" name="service" [(ngModel)]="participantData.service">
+    </mat-form-field>
+
+    <mat-form-field >
+      <input matInput placeholder="Social media account (@username)" name="username" [(ngModel)]="participantData.username">
```
Comment (Ldhatter) [inline angular-client/src/app/components/new-participant/new-participant.component.html]: can we automate that the "@" is already in the input field and the user doesnt have to write it?

Comment (adriannadiaz) [reply angular-client/src/app/components/new-participant/new-participant.component.html]: It really depends on the platform. But it doesn’t have to be.

**Pull request merged**